docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -134,6 +134,13 @@ Wildcard expressions are not accepted.
 --
 end::expand-wildcards[]
 
+tag::field_statistics[]
+`field_statistics`::
+(Optional, boolean) Whether document count, sum of document frequencies, and 
```

Comment:
For boolean values, we've been trying to use the following format:

> If `true`, ...

Idea is courtesy of @KOTungseth.